src/wp-includes/block-template-utils.php

```diff
@@ -224,14 +224,21 @@ static function ( $item ) {
  * @return string[] A list of paths to all template part files.
  */
 function _get_block_templates_paths( $base_directory ) {
+	static $wp_path_list = array();
```

Comment:
Not blocker. Could you update the variable name as core don't use `wp_` prefix before variables?

I.e.
```suggestion
	static $template_path_list = array();
```

Reply:
Made sense to me. Changed the variable name.